eng/pipelines/libraries/base-job.yml

```diff
@@ -65,19 +69,30 @@ jobs:
         - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
           - _msbuildCommonParameters: /p:OfficialBuildId=$(Build.BuildNumber)
 
+        - _coreClrArtifactName: ''
+        - _coreClrDownloadPath: ''
+        - _coreClrOverridePathArg: ''
+
+        - ${{ if eq(parameters.useLiveCoreClr, true) }}:
+          - _coreClrDownloadPath: '$(Build.SourcesDirectory)/artifacts/transport/coreclr'
+          - _coreClrArtifactName: 'CoreCLRProduct_${{ parameters.osGroup }}${{ parameters.osSubgroup }}_${{ parameters.archType }}_${{ parameters.buildConfig }}'
+          - _coreClrOverridePathArg: ' /p:CoreCLROverridePath=$(_coreClrDownloadPath)'
+
         # Windows variables
         - ${{ if eq(parameters.osGroup, 'Windows_NT') }}:
           - _msbuildCommand: powershell -ExecutionPolicy ByPass -NoProfile eng\common\msbuild.ps1 -warnaserror:0 -ci
           - _runtimeOSArg: /p:RuntimeOS=win10
+          - _archiveExtension: '.zip'
```

Comment:
Why do we need this? This should already be defined in `xplat-setup.yml`

Reply:
Ahh, you're right, this should be no longer necessary with all your refactorings, will remove, thank you!